version/version.go

```diff
@@ -4,7 +4,7 @@ import "github.com/hashicorp/packer-plugin-sdk/version"
 
 var (
 	// Version is the main version number that is being run at the moment.
-	Version = "1.1.7"
+	Version = "1.2.1"
```

Comment:
I know that with the tag goreleaser will override the version of the binary. But if you check out this plugin @v1.2.0 and build locally the version will be 1.2.2-dev which might be confusing. Not worth changing at this moment since this is just my opinion but something to think about.

Reply:
Oh yeah so, the tag will be on the commit what bumps the version to 1.2.0 and removes the `dev` suffix, I just bundled both the version update for the release and the preparation for the next one to make a single PR to release and prepare.
Would it be clearer if I split this in two different PRs?